app/presenters/step_by_step_page_presenter.rb

```diff
@@ -103,6 +103,26 @@ def secondary_links_settings
     }
   end
 
+  def tags_section
+    {
+      borderless: true,
+      id: "tags",
+      title: "Tags",
+      items: [
+        {
+          field: "Topics",
+          edit: {
+            link_text: "Edit",
+            href: "#{Plek.find('content-tagger', external: true)}/taggings/#{step_by_step_page.content_id}",
+            data_attributes: {
+              gtm: "edit-topics",
+            },
+          },
+        },
+      ],
+    }
```

Comment:
I think this can just be: 

```
    {
      borderless: true,
      id: "tags",
      title: "Tags",
      edit: {
        href: "#{Plek.find('content-tagger', external: true)}/taggings/#{step_by_step_page.content_id}",
          data_attributes: {
            gtm: "edit-topics",
           },
         },
      ],
    }
```

Reply:
This will give a layout like this:
![Screen Shot 2019-10-11 at 15 20 33](https://user-images.githubusercontent.com/6329861/66659103-b75b8f00-ec3a-11e9-9f1e-87583e31ae94.png)

Are we happy just having the word "Tags", or should it be "Topics"?

Reply:
It does say "Tags" in the card, so I would leave it as is.

Reply:
Do you really need to nest "edit" inside "items"?

Reply:
No, I haven't yet pushed the code that rendered the above screenshot

Reply:
Pushed the updated code

Reply:
Oops! Sorry.